<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterConfiguration</class>
 <widget class="QWidget" name="FilterConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>342</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>16</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Single Value Filters</string>
   </property>
   <widget class="QSpinBox" name="singleVal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-999999999</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="minFilter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>39</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="equalFilter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>49</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Equal</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="differentFilter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Different</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="maxFilter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Range Filters</string>
   </property>
   <widget class="QSpinBox" name="maxVal">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-999999999</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="inRangeFilter">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>26</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>In Range</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="outRangeFilter">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>66</y>
      <width>94</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Outside Range</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QSpinBox" name="minVal">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-999999999</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="setFilterButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Filter</string>
   </property>
  </widget>
  <action name="actionUse_Texture_Rectangle">
   <property name="text">
    <string>Use Texture Rectangle</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>displayTypeChanged(int)</slot>
  <slot>outputFormatChanged(int)</slot>
  <slot>textureRectangleChecked(int)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
